k8up/Makefile

```
@@ -0,0 +1,48 @@
+MAKEFLAGS += --warn-undefined-variables
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DEFAULT_GOAL := help
+.DELETE_ON_ERROR:
+.SUFFIXES:
+
+# Git-treeish of the K8up archive version to fetch
+K8UP_VERSION := master
+CRD_FILLE    := templates/crds.yaml
+
+SED_CMD := gsed
+
+# TODO Use the aggregated CRDs that will be attached to a release.
+# This will get rid of creating and removing the temporary `crds` folder.
+#
+# This is made phony on purpose. We do not have a local file that would trigger a rebuild.
+.PHONY: templates/crds.yaml
+templates/crds.yaml: ## Update the CRDs by fetching them from the k8up repository.
+	mkdir -p crds
+	curl -sL https://github.com/vshn/k8up/archive/$(K8UP_VERSION).tar.gz | tar -xj --directory crds --strip-components=4 k8up-master/config/crd/apiextensions.k8s.io/
+	echo -n "{{ if Capabilities.KubeVersion.Major == 1 && Capabilities.KubeVersion.Minor < 16 }}" > $@
```

I think we should give the user more flexibility to install / maintain charts. Here, it's missing at least the a `.Values.crd.create` boolean flag:
```suggestion
	echo -n "{{ if Capabilities.KubeVersion.Major == 1 && Capabilities.KubeVersion.Minor < 16 && .Values.crd.create || .Values.crd.forceLegacyCrd }}" > $@
```